### PR TITLE
Redesign/270 contact page

### PR DESCRIPTION
**Pull request by cdufland**
Resubmit to merge with proper branch

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -217,9 +219,9 @@ a.dropdown-item:active {
 02. HEADER
 *===================================*/
 header {
-  padding: 25px 65px;
+  padding: 15px 65px;
```
Comment (RyanRussell00) [inline assets/css/styles.css:222]: Don't use absolute sizing. Please change this to relative. 
https://www.w3schools.com/CSSref/css_units.asp

File: assets/css/styles.css [hunk 2]
```diff
@@ -239,7 +241,7 @@ header.active {
 }
 
 .navbar .navbar-brand img {
-  max-width: 140px;
+  max-width: 75px;
```
Comment (RyanRussell00) [inline assets/css/styles.css:244]: Change to relative sizing

File: assets/css/styles.css [hunk 3]
```diff
@@ -297,12 +299,12 @@ header.active {
 }
 
 header.nav-fixed {
-  padding: 10px 15px;
-  background-color: #124f8f;
+  padding: 10px 15px;
```
Comment (RyanRussell00) [inline assets/css/styles.css:302]: Change to relative sizing

File: assets/css/styles.css [hunk 4]
```diff
@@ -297,12 +299,12 @@ header.active {
...
 header.nav-fixed {
-  padding: 10px 15px;
-  background-color: #124f8f;
+  padding: 10px 15px;   
+  background-color: var(--ACM-blue);
 }
 
 header.nav-fixed .navbar-brand img {
-  max-width: 102px;
+  max-width: 75px;
```
Comment (RyanRussell00) [inline assets/css/styles.css:307]: Change to relative sizing

File: assets/css/styles.css [hunk 5]
```diff
@@ -535,12 +537,16 @@ max-width: 1440px
 ---------------------------------------------------*/
 @media only screen and (max-width: 1440px) {
   header {
-    padding: 25px 15px;
+    padding: 15px 15px;
```
Comment (RyanRussell00) [inline assets/css/styles.css:540]: Change to relative sizing

File: assets/css/styles.css [hunk 6]
```diff
@@ -535,12 +537,16 @@ max-width: 1440px
...
   }
 
   .navbar-expand-lg .nav_btn>li {
     padding: 0;
   }
+  .navbar .navbar-brand img {
+    max-width: 75px;
```
Comment (RyanRussell00) [inline assets/css/styles.css:547]: Change to relative sizing

File: assets/css/styles.css [hunk 7]
```diff
@@ -552,7 +558,8 @@ max-width: 1199px
   }
 
   .navbar .navbar-brand img {
-    max-width: 109px;
+    max-width: 75px;
```
Comment (RyanRussell00) [inline assets/css/styles.css:561]: Change to relative sizing

File: assets/css/styles.css [hunk 8]
```diff
@@ -597,7 +604,8 @@ max-width: 991px
   }
 
   .navbar .navbar-brand img {
-    max-width: 140px;
+    max-width: 50px;
```
Comment (RyanRussell00) [inline assets/css/styles.css:607]: Change to relative sizing

File: assets/css/styles.css [hunk 9]
```diff
@@ -670,7 +678,7 @@ max-width: 480px
 ---------------------------------------------------*/
 @media only screen and (max-width: 480px) {
   .navbar-brand img {
-    max-width: 200px;
+    max-width: 50 px;
```
Comment (RyanRussell00) [inline assets/css/styles.css:681]: Change to relative sizing

File: pages/contact.html [hunk 10]
```diff
@@ -33,14 +33,20 @@
...
 <body>
   <!-- site header -->
   <header id="site-header" class="fixed-top">
     <nav class="navbar navbar-expand-lg navbar-dark">
-      <a class="navbar-brand" href="index.html">
-        <span class="fa fa-shield"></span> Captivate
+      <a class="navbar-brand" href="../../index.html">
+        <span>
+          <img src="../assets/images/logos/favicons/favicon.ico">
+        </span> UWB ACM
```
Comment (RyanRussell00) [inline pages/contact.html:49]: I have made a new issue for redesigning the header #287 

You can leave these header changes for the time being, but please don't make any more changes.

File: pages/contact.html [hunk 11]
```diff
@@ -89,102 +90,92 @@
...
+        <h5 class="section-title">Interested in working with us?</h5>
+        <p class= "text-center"> Are you an alum who wants to participate in our events? Email us to see how you can make difference with the UWB ACM. </p>
+          </div>
       </div>
     </div>
   </section>
-
   <section class="contact-form">
     <div class="container">
       <div class="contacts12-main">
```
Comment (RyanRussell00) [inline pages/contact.html:153]: The `<section>`, and top 2 `<div>` are not being closed. If you look at the website you'll see that the footer is broken.

File: pages/contact.html [hunk 12]
```diff
@@ -147,24 +147,15 @@ <h5 class="section-title">Interested in working with us?</h5>
...
-        </div>
-      </div>
-    </div>
-  </section>
+  </section>  
 <!-- Begin Mailchimp Signup Form -->
 <link href="//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" rel="stylesheet" type="text/css">
 <div id="mc_embed_signup">
   <form action="https://uwbacm.us16.list-manage.com/subscribe/post?u=8773dffc3f528b0b18fd7aef8&amp;id=7a3e5f69db"
     method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_"
```
Comment (RyanRussell00) [inline pages/contact.html:155]: Needs to be `target="_blank"`

**Comment by cdufland** [comment]
done..again...third times the charm? lol

**Comment by RyanRussell00** [comment]
@cdufland There are conflicts on this branch to `redesign/master` can you please handle these conflicts then it should be good to merge!

**Pull request merged**